coalib/parsing/DefaultArgParser.py

```diff
@@ -155,6 +155,13 @@ def default_arg_parser(formatter_class=None):
         '--no-autoapply-warn', const=True, action='store_const',
         help='turn off warning about patches not being auto applicable')
 
+    config_group.add_argument('-g',
+                              '--generate-config', nargs=2,
```

Comment:
Why is there a limit on `nargs`? Shouldn't this be `+` instead?

Reply:
It should probably accept a list of `language:style_name` strings. A project could be have more than one language.

Reply:
Just noticed https://github.com/coala/coala/pull/6049#discussion_r312834186, anyway I think `language:style_name` syntax is a better choice for readability reasons.